Queue using Stack.py

```diff
@@ -0,0 +1,42 @@
+""""// Time Complexity : 26ms
```

Comment:
O(1) for all operations, please don't mention runtime here. Only mention Big O notation for your solutions